tests/e2e/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/oasisprotocol/oasis-sdk/tests/e2e
+
+go 1.15
+
+// Keep these in sync with oasis-core!
+replace (
+	github.com/coreos/etcd => github.com/coreos/etcd v3.3.25+incompatible
+	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
+
+	github.com/tendermint/tendermint => github.com/oasisprotocol/tendermint v0.34.8-oasis1
+	golang.org/x/crypto/curve25519 => github.com/oasisprotocol/ed25519/extra/x25519 v0.0.0-20210127160119-f7017427c1ea
+	golang.org/x/crypto/ed25519 => github.com/oasisprotocol/ed25519 v0.0.0-20210127160119-f7017427c1ea
+)
+
+require (
+	github.com/oasisprotocol/oasis-core/go v0.2100.1
+	github.com/oasisprotocol/oasis-sdk/client-sdk/go v0.0.0-20210319174144-fdda3859da1f
```

Comment:
Can this depend on `../../client-sdk/go`? Maybe using a replace directive if it doesn't work directly? As we want to test the correct version.

Reply:
Done :)